domains/akris.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "AKris15",
+    "email": "herogupta626@gmail.com"
+  },
+  "record": {
```

```suggestion
  "records": {
```
invalid Domain Structure please read -> https://docs.is-a.dev/domain-structure/

---

made the appropriate changes